ecms_base/features/custom/ecms_projects/config/install/node.type.project.yml

```diff
@@ -0,0 +1,16 @@
+langcode: en
+status: true
+dependencies:
+  module:
+    - menu_ui
+third_party_settings:
+  menu_ui:
+    available_menus: {  }
+    parent: ''
+name: Project
+type: project
+description: 'Manage construction projects for DOT '
```

Comment:
Extra space at the end of this description.

Reply:
Resolved, and made it generic to "agencies."